docs/en/modules/file.md

```diff
@@ -144,13 +144,17 @@ print("\nFile system info:\nTotal : "..total.." (k)Bytes\nUsed : "..used.." (k)B
 Lists all files in the file system.
 
 #### Syntax
-`file.list()`
+`file.list([pattern])`
 
 #### Parameters
 none
 
 #### Returns
-a Lua table which contains the {file name: file size} pairs
+a Lua table which contains all {file name: file size} pairs, if no pattern
+given.  If a pattern is given, only those file names matching the pattern
+(interpreted as a traditional Lua pattern, not, say, a UNIX shell glob) will be
```

Comment:
Linking to https://www.lua.org/pil/20.2.html or http://lua-users.org/wiki/PatternsTutorial here?

Reply:
Sure. :+1: